src/transaction/utils.ts

```diff
@@ -12,6 +12,15 @@ export type ApiMemo = {
   MemoFormat?: string
 }
 
+export type TransactionJSON = {
+  Account: string,
+  TransactionType: string,
+  Memos?: {Memo: ApiMemo}[],
+  Flags?: number,
+  Fulfillment?: string,
```

Comment:
It's weird to me that `Fulfillment` (an EscrowFinish-specifie field) is here, but the other [common fields](https://developers.ripple.com/transaction-common-fields.html) are not.

Is it because the required transaction cost is higher if `Fulfillment` is provided?

Reply:
Yes, exactly